source/partials/links/_link_internal.html.slim

```diff
@@ -3,8 +3,9 @@ ruby:
   label ||= cta_label(item)
   white_outline ||= nil
   border = white_outline ? "box-shadow: inset 0 0 0 2px white" : ""
+  page_link_path = item.anchor.present? ? page_path_with_anchor(item.link, item.anchor.present? ? "##{item.anchor}" : nil) : page_path(item.link)
```

Comment:
Is this needed for the video player or is this change unrelated?

Anyway, can't we do something like this?
```suggestion
page_link_path = item.anchor&.present? ? page_path_with_anchor(item.link, "##{item.anchor}") : page_path(item.link)

```

Reply:
this is needed for, internal link anchors, problably should be in another pr, but is used for the cookie consent shortcut related to video.